pom.xml

```diff
@@ -318,6 +318,14 @@
               <pattern>com.fasterxml.jackson</pattern>
               <shadedPattern>com.spotify.docker.client.shaded.com.fasterxml.jackson</shadedPattern>
             </relocation>
+            <relocation>
+              <pattern>org.ow2.asm</pattern>
+              <shadedPattern>com.spotify.docker.client.shaded.org.ow2.asm</shadedPattern>
+            </relocation>
+            <relocation>
+              <pattern>com.github.jnr</pattern>
```

Comment:
I realized that this is the wrong relocation pattern: while the artifact's organization name is `com.github.jnr`, the JNR classes are under the `jnr.*` namespace.